app/views/season_admin/index.html.erb

```diff
@@ -3,7 +3,22 @@
 <h1>Időszakok állítása</h1>
 <div id="content-main" class="uk-panel uk-panel-box">
   <div class="uk-width-medium-1-2 uk-container-center uk-overflow-container">
+    <div class="uk-margin">
+      <table>
+        <tr>
+          <td><%= button_to "Elöző félév kiválasztása", {method: :post, action: "previous"},
```

Comment:
Button_to is rearly used because is parameter list is very unusall.
It would be batter to use link_to + route helper, something like this:
```suggestion
        <td><%= link_to "Elöző félév kiválasztása", previous_semester_path, {method: :post,
                            class: 'uk-button uk-button-danger uk-margin-small uk-margin-left'} %></td>
```